### DISCUSSION TITLE
Search by IMDB ID or TVDB ID

### DISCUSSION THREAD

**Discussion by austinbarberrepo** [sct/overseerr#1135] [Help] [answered]
Is it possible to search by using the IMDB or TVDB ID like in radarr and sonarr? I tried to search for a series called "Them" and a ton came back. Here is the TVDB link with the ID - https://www.thetvdb.com/series/them-covenant. If I need to put in a request I can. Just wanted to make sure I wasn't doing something wrong. Thank you!

**Comment by TheCatLady** [comment, marked as answer]
Hi @abarber229!  Our search is handled by TMDb, so we don't currently support searching for items by ID.

However, we do plan to add additional filtering options in the future, which would help alleviate issues like this.

For now, if you want to navigate directly to a page for a specific movie/series, you can use `{your Overseerr URL}/movie/{movie TMDb ID}` or `{your Overseerr URL}/tv/{series TMDb ID}`.

**Reply by austinbarberrepo** [reply]
Sounds great! Thank you for the workaround.

**Reply by dcglabs** [reply]
As of 1.29.0 you can search by ID.
`You can now search by IMDb, TVDB, or TMDb ID directly in the search box by appending the respective service in front of the ID (like tmdb:1234)`
#2082

**Reply by JediKnightTiggr** [reply]
Are there any plans/workarounds to allow TVDB Episode Ordering/Checklist for viewing what's in Plex? Series in particular.
Plex uses an amalgamation of different sources, although I've noticed it tends to lean heavier on TVDB, and so I've just used FileBot for TVDB structuring (except a handful that run off TMDB episode combinations)
Sonarr ONLY uses TVDB for their structure. 
It's annoying having to manually check off TMDB Season 2 when it's equivalent to a complete TVDB Season 1.

**Reply by wormangel** [reply]
Building up on @JediKnightTiggr's request, and also referencing [this old issue](https://github.com/sct/overseerr/issues/2648) - the ecosystem of Plex, the *arr apps, and the remaining tools around it (where Overseerr fits) is great precisely because of how they play well with eachother. The answer in the other issue says nothing can be done, but I disagree - Overseerr could change to work with different sources (giving the option to the user to select or not, maybe "smartly" use the same source as radarr/sonarr if those expose that somewhere in their APIs, etc).

If that's undesireable or absolutely not going to happen, would you be able to provide some pointers in high level (and/or with more precise pointers to the codebase) of how the metadata integration (with TMDb) currently works, so we can maybe open PRs or even fork the project?

Thanks in advance from a fan of Overseerr :)
